### PR TITLE
Can set showCoverageOnHover for vector layer

### PR DESCRIPTION
**Pull request by michaelpnelson**
This adds the ability to toggle the "showCoverageOnHover" property on a vector layer.

package.json is updated to require smk version 1.0.6 or greater as the ability to read the showCoverageOnHover property in a vector layer was added in that version.

Docs are updated to reflect UI changes as well as make text more consistent.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
       "version": "1.0.1",
```
Comment (dgboss) [inline package-lock.json:9]: I imagine we're going to be releasing 1.0.2 pretty soon. You might want to consider updating the version in package.json and package-lock.json to 1.0.2 as part of this PR.

File: smk-edit/static/components/edit-item.css [hunk 2]
```diff
@@ -1,3 +1,6 @@
+.modal.edit-item .modal-content {
+    height: calc(100vh - 56px);
```
Comment (dgboss) [inline smk-edit/static/components/edit-item.css:2]: Apparently this css isn't quite enough to solve all the issue with the UI. When I run the smk editor app, there is some UI that isn't visible. The horizontal line below metadata URL and the bottom arrow control of the scroll bar.

image

**Review by dgboss — CHANGES_REQUESTED** [review]

File: docs/edit-layer.md [hunk 3]
```diff
@@ -100,11 +100,11 @@ From the WMS Details panel, you can modify the following attributes:
 
 #### Style Name
 
-`Style Name` Is the name of the WMS Service layer style you are presenting to your users. This should not be modified unless you are certain of a change to source layer styles (confirmed by a review of the WMS Service GetCapabilities). Usually, it's easier to simply re-add the layer.
+`Style Name` is the name of the WMS Service layer style you are presenting to your users. This should not be modified unless you are certain of a change to source layer styles (confirmed by a review of the WMS Service GetCapabilities). Usually, it's easier to simply re-add the layer.
 
 #### Metadata URL
 
-`Metadata URL` is A supplied URL for the layers metadata. By default this is empty for WMS layers, but can be supplied if desired. This is useful for providing users with links to data sources or specific external resources related to the data displayed in your application.
+`Metadata URL` is a supplied URL for the layers metadata. By default this is empty for WMS layers, but can be supplied if desired. This is useful for providing users with links to data sources or specific external resources related to the data displayed in your application.
```
[the remark this answers is not in the document]
Comment (michaelpnelson) [reply docs/edit-layer.md:107]: Done

File: docs/edit-layer.md [hunk 4]
```diff
@@ -132,33 +133,39 @@ Note that the Vector Details panel is the same if you've imported vector data di
 
 #### Opacity
 
-`Opacity` adjust how transparent the layer is by default. Making a layer 100% Opaque will prevent users from seeing feaures covered by this layer. Setting a layer to 0% Opaque will prevent it from displaying on the map at all, but it will still be identifiable and selectable (just invisible).
+`Opacity` adjusts how transparent the layer is by default. Making a layer 100% Opaque will prevent users from seeing feaures covered by this layer. Setting a layer to 0% Opaque will prevent it from displaying on the map at all, but it will still be identifiable and selectable (just invisible).
```
[the remark this answers is not in the document]
Comment (michaelpnelson) [reply docs/edit-layer.md:136]: Done

File: docs/edit-layer.md [hunk 5]
```diff
@@ -132,33 +133,39 @@ Note that the Vector Details panel is the same if you've imported vector data di
...
+#### Show Cluster Bounds on Hover 
+
+`Show Cluster Bounds on Hover` shows the bounds of a cluster's markers when you mouse over the cluster.  
+
+> __Note:__ This option only appears when `Show Features in Clusters` is selected for `How should the Data be Displayed?`.
+
+#### Data URL
 
-`Show Features as a Heatmap` Will remove your default styling and convert the data into a heatmap display. This is very useful for point data sets.
+`Data URL` is the URL location for your vector data. If you've imported a vector file into your application, this will be a local path to your applications source folder. Do not modify this URL unless you are sure you want to direct the application to search a specific location for the vector data.
```
[the remark this answers is not in the document]
Comment (michaelpnelson) [reply docs/edit-layer.md:164]: Done

File: docs/edit-layer.md [hunk 6]
```diff
@@ -132,33 +133,39 @@ Note that the Vector Details panel is the same if you've imported vector data di
...
 
 #### Metadata URL
 
-`Metadata URL` is A supplied URL for the layers metadata. By default this is empty for WMS layers, but can be supplied if desired. This is useful for providing users with links to data sources or specific external resources related to the data displayed in your application.
+`Metadata URL` is a supplied URL for the layers metadata. By default this is empty for WMS layers, but can be supplied if desired. This is useful for providing users with links to data sources or specific external resources related to the data displayed in your application.
```
[the remark this answers is not in the document]
Comment (michaelpnelson) [reply docs/edit-layer.md:168]: Done

File: package-lock.json (hunk 1 above)
Comment (michaelpnelson) [reply package-lock.json:9]: I'd planned to increment to 1.0.2 in a separate commit, but it makes sense to do it in this commit as these changes are the last meant to be included in the 1.0.2 release.

File: smk-edit/static/components/edit-item.css (hunk 2 above)
Comment (dgboss) [reply smk-edit/static/components/edit-item.css]: I have another option that seems to work well and doesn't break anything that I can see. It also allows us to maintain the original order of the inputs (ie. no need to move the Data Url input. Note the v-bind:style="{position:'relative'}" attribute on the parent div of the checkbox in the following code snippet:
`
<div>
    <div class="row">
        <input-textarea class="col s12"
            v-model="dataUrl"
        >Data URL</input-textarea>
    </div>
    <div class="row">
        <input-select class="col s12"
            v-model="displayMode"
        >How should the data be displayed?
            <template v-slot:options>
                <option value="raw">Show features as styled</option>
                <option value="cluster">Show features in clusters</option>
                <option value="heatmap">Show features as a heatmap</option>
            </template>
        </input-select>
    </div>
    <div class="row" v-bind:style="{position:'relative'}">
        <input-checkbox v-if="displayMode === 'cluster'" class="col s12"
            v-model="showCoverageOnHover"
        >Show the bounds of a cluster on hover</input-checkbox>
    </div>
</div>
`

Comment (michaelpnelson) [reply smk-edit/static/components/edit-item.css]: Great suggestion! I've applied it and confirmed there's no longer a jump (even after restoring the original ordering of widgets) in Chrome, Firefox and Safari.

**Review by dgboss — APPROVED** [review]

**Pull request merged**